packages/opentelemetry-exporter-collector/src/platform/browser/util.ts

```diff
@@ -72,7 +72,7 @@ export function sendWithXhr(
         onSuccess();
       } else {
         const error = new collectorTypes.CollectorExporterError(
-          xhr.responseText,
+          `Failed to export with XHR (status: ${xhr.status})`,
```

Comment:
What is the reason for this change?

Reply:
Depending on what the collector report (which itself depend on upstream exporters) you can have a empty text. I prefered to opt for this generic text so user get an error message every time.

Reply:
`xhr.status` already is passed to `CollectorExporterError`. Maybe it is better to have `xhr.responseText || "Failed to export with XHR"`?